amy/settings.py

```diff
@@ -190,10 +191,20 @@
 # Database
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 
+if DEBUG:
+    DB_FILENAME = os.environ.get('AMY_DB_FILENAME', 'db.sqlite3')
+else:
+    try:
+        DB_FILENAME = os.environ['AMY_DB_FILENAME']
+    except KeyError as ex:
+        raise ImproperlyConfigured(
+            'You must specify AMY_DB_FILENAME environment variable '
+            'when DEBUG is False.') from ex
```

Comment:
Sorry, I rarely see "raise ex1 from ex2", can you please explain to me in this thread what it does?